objetos/funcionarios/Funcionario.java

```diff
@@ -30,4 +32,25 @@ public double getSalario() {
     public void setSalario(double salario) {
         this.salario = salario;
     }
+
+    public NiveisCargoFuncionario getNivelCargo() {
+        return nivelCargo;
+    }
+
+    public boolean setNivelCargo(String cargo) {
+        try {
+            this.nivelCargo = NiveisCargoFuncionario.valueOf(cargo);
+            return true;
+        } catch (IllegalArgumentException e) {
+            System.out.println("Nível de cargo invalido.");
+            return false;
+        }
+    }
+
+    public void imprimirNiveisCargoFuncionario() {
+        NiveisCargoFuncionario[] niveisCargo = NiveisCargoFuncionario.values();
+        for (NiveisCargoFuncionario nivel : niveisCargo) {
```

Comment:
O for pode ser alterado para um fori, para exibir números ao lado de cada tipo de status diferente.
Exemplo:

1. INICIANTE
2. EXPERIENTE
3. AVANCADO